sklearn/metrics/_plot/det_curve.py

```diff
@@ -265,6 +276,13 @@ def from_predictions(
         >>> plt.show()
         """
         check_matplotlib_support(f"{cls.__name__}.from_predictions")
+
+        target_type = type_of_target(y_true)
+        if target_type != "binary":
+            raise ValueError(
+                f"The target y is not binary. Got {target_type} type of target."
+            )
```

Comment:
Do we need this check here, since we do the check in `det_curve`?

https://github.com/scikit-learn/scikit-learn/blob/845b1fac9e8bcfc146e7b38119ce6c500e00a10b/sklearn/metrics/_ranking.py#L310-L314